### ISSUE TITLE
Make sure the user ID/token is included in the data sent when sending a message

### ISSUE THREAD

**Issue by webinista** [department-of-veterans-affairs/messaging-fe#83]
Need to include "from" data -- user token? name? -- with the create message request.

Blocked by authentication.

**Comment by robbiethegeek** [comment]
isn't this data that would be pulled from the data store?

**Comment by webinista** [comment]
Yeah, probably. The point was more to account for the task. Updated the title to be more accurate.

**Comment by webinista** [comment]
Duplicate of department-of-veterans-affairs/kudos-team#215
